aggregator/core/pom.xml

```diff
@@ -38,5 +38,6 @@ under the License.
     <module>../../../core/resolver</module>
     <module>../../../core/resolver-ant-tasks</module>
     <module>../../../core/wrapper</module>
+    <!--module>3.x</module-->
```

Comment:
Would it make sense to move this to a profile, like we did with studies some time ago? BTW: Is the `3.x/pom.xml` missing?